tfhe/src/shortint/oprf.rs

```diff
@@ -117,6 +121,80 @@ pub(crate) mod test {
         a * a
     }
 
+    #[test]
+    // This test is seeded which prevents flakiness
+    // The noise added by the KS and the MS before the PRF LUT evaluation can make this test fail
+    // if the seeded input is close to a boundary between 2 encoded values
+    // Using another KS key can, with a non-neglibgible probability,
```

Comment:
does it mean that if the KS key generated had different encryption noise we would see different results ?

Reply:
It's possible, but most of the time, after decryption and decoding, we'd get the same result

Reply:
I just want to make sure I understand your comment :)

Reply:
assuming we use the same secret keys, ha I think I know why your key has a hamming weight ratio that is 1 not 1/2, try with 1/2 ?

Reply:
Fixed hamming weight